src/scriptworker/constants.py

```diff
@@ -373,6 +373,24 @@
                             "project:releng:treescript:action:push": "all-production-branches-and-autoland",
                             "project:releng:microsoftstore:beta": "beta",
                             "project:releng:microsoftstore:release": "release",
+                            # Note that the repositories in the scope are identifiers in Lando
+                            # and not necessarily individual repositories. For example, the Lando
+                            # "beta" repository is ultimately the `beta` branch of
+                            # https://github.com/mozilla-firefox/firefox.
+                            "project:releng:lando:repo:autoland": "autoland",
+                            "project:releng:lando:repo:nightly": "nightly",
+                            "project:releng:lando:repo:beta": "beta",
+                            "project:releng:lando:repo:release": "release",
+                            "project:releng:lando:repo:esr115": "esr115",
+                            "project:releng:lando:repo:esr128": "esr128",
+                            "project:releng:lando:actions:android_l10n_import": "autoland",
+                            "project:releng:lando:actions:android_l10n_sync": "beta",
+                            "project:releng:lando:actions:merge_day": "merge-day-branches",
+                            "project:releng:lando:actions:l10n_bump": "l10n-bump-branches",
```

Comment:
Should all these actions be restricted, or is there a use case for running certain actions on try, and only restrict the `lando:repo:*` scopes?